crates/static-analysis-kernel/src/analysis/ddsa_lib/common.rs

```diff
@@ -8,12 +8,48 @@ use std::fmt::Debug;
 
 #[derive(Debug, thiserror::Error)]
 pub enum DDSAJsRuntimeError {
+    #[error("expected `{name}` to exist within the v8 context")]
+    VariableNotFound { name: String },
+    #[error("type should be \"{expected}\", not \"{got}\"")]
+    WrongType {
+        expected: &'static str,
+        got: &'static str,
+    },
+    #[error("invalid value for `{identifier}`: expected {expected}")]
+    InvalidValue {
+        identifier: &'static str,
+        expected: &'static str,
+    },
     /// A (shorthand) generic error used to fail a test. The test itself will have the appropriate context.
     #[cfg(test)]
     #[error("cfg(test): unspecified")]
     Unspecified,
 }
 
+/// Loads a global [`v8::Function`] from the provided scope, returning an error if either the
+/// identifier doesn't exist, or if it doesn't refer to a function.
+pub fn load_function(
+    scope: &mut HandleScope,
+    identifier: &str,
+) -> Result<v8::Global<v8::Function>, DDSAJsRuntimeError> {
+    let ctx = scope.get_current_context();
+    let global = ctx.global(scope);
+
+    let s_func_name = v8_interned(scope, identifier);
+    let id_value = global.get(scope, s_func_name.into()).ok_or_else(|| {
+        DDSAJsRuntimeError::VariableNotFound {
+            name: identifier.to_string(),
+        }
+    })?;
+    let func = v8::Local::<v8::Function>::try_from(id_value).map_err(|_| {
+        DDSAJsRuntimeError::WrongType {
+            expected: "function",
+            got: id_value.type_repr(),
+        }
+    })?;
+    Ok(v8::Global::new(scope, func))
+}
+
```

Comment:
Seems to be only used in tests. Do you plan to use it to load the new runtime or it's purely done for testing? If the latter, maybe move it into tests?